### PR TITLE
Feature expose studytype

### PR DESCRIPTION
**Pull request by SnehaCRao**

### CODE REVIEW

File: dynatademand/schemas/request/body/create_line_item.json [hunk 1]
```diff
@@ -45,12 +45,7 @@
     "deliveryType": {
       "type": "string",
       "default": "BALANCED",
-      "description": "The plan on how responses will flow into the survey.",
-      "enum": [
```
Comment (dynata-bradley) [inline dynatademand/schemas/request/body/create_line_item.json:49]: Why delete this?

File: dynatademand/schemas/request/body/create_project.json [hunk 2]
```diff
@@ -118,12 +132,7 @@
           "deliveryType": {
             "type": "string",
             "default": "BALANCED",
-            "description": "The plan on how responses will flow into the survey.",
-            "enum": [
```
Comment (dynata-bradley) [inline dynatademand/schemas/request/body/create_project.json:122]: Why delete this?

File: dynatademand/api.py [hunk 3]
```diff
@@ -158,6 +158,9 @@ def get_countries(self, **kwargs):
         )
         return self._api_get('/countries', kwargs)
 
+    def get_study_metadata(self):
+        return self._api_get('/studyMetadata')
```
Comment (dynata-bradley) [inline dynatademand/api.py:162]: Is this metadata for a single project or all projects?

Comment (SnehaCRao) [reply dynatademand/api.py:162]: It is for all the projects
https://developers.dynata.com/demand-api-reference/data_endpoints/metadata/study-metadata

File: dynatademand/schemas/request/body/create_project.json (hunk 2 above)
Comment (SnehaCRao) [reply dynatademand/schemas/request/body/create_project.json:122]: It is been replaced with string and validated for the input. The enum values are no longer being used in the implementation.

File: dynatademand/schemas/request/body/create_line_item.json (hunk 1 above)
Comment (SnehaCRao) [reply dynatademand/schemas/request/body/create_line_item.json:49]: It is been replaced with string and validated for the input. The enum values are no longer being used in the implementation.

File: dynatademand/api.py (hunk 3 above)
Comment (DynataRidley) [reply dynatademand/api.py:162]: Can we add a test for this function? I know it's a simple API call, but we have tests for the other simple API calls as well for coverage.

Comment (SnehaCRao) [reply dynatademand/api.py:162]: @DynataRidley, added test case for the studyMetaData endpoint.

**Pull request merged**